lib/player.js

```diff
@@ -1352,6 +1352,12 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
       const extension = shaka.media.ManifestParser.getExtension(uri);
       mimeType = shaka.Player.SRC_EQUAL_EXTENSIONS_TO_MIME_TYPES_[extension];
     }
+    if (!mimeType && uri.includes('.ism') && uri.includes('format=m3u8')) {
```

Comment:
Should this perhaps be `extension == '.ism'` instead of `uri.includes('.ism')`?

Reply:
The URLs are from Microsoft, here you can see some examples: https://learn.microsoft.com/en-us/azure/media-services/previous/media-services-deliver-content-overview#URLs

As such it is not an extension and `shaka.media.ManifestParser.getExtension(uri);` returns `''`

![image](https://user-images.githubusercontent.com/8983024/194403646-32dd843d-6ef3-4ad4-aa9b-69ae1d391117.png)